contracts/colony/ColonyExpenditure.sol

```diff
@@ -77,7 +77,7 @@ contract ColonyExpenditure is ColonyStorage {
     stoppable
     authDomain(_permissionDomainId, _childSkillIndex, expenditures[_id].domainId)
     expenditureExists(_id)
-    expenditureActive(_id)
+    expenditureDraft(_id)
```

Comment:
Arbitration can transfer the expenditure via `setExpenditureState` regardless of expenditure state. That's the preferred interface for all arbitration actions, rather than having separate permissioned functions.

Reply:
And you don't think the owner should be allowed to do so?

Reply:
It's a good question. Consider the scenario where the owner put down a stake to create the expenditure. Does transferring to another user mean that the owner gets their stake back? Does the new user have to put down a stake?

I would be comfortable allowing the owner to transfer as long as the stake remained with the initial user, which is doable by mapping `expenditureId => user` in the staking extension.

Reply:
I mean, we're talking about the primitive here. In the scenario you describe, the owner of the expenditure is the extension.

These changes are being done so that this primitive is, in some sense, 'useful' on its own, which suggests we should ask product?

Reply:
Personally, though, I would lean towards allowing the owner to do as much as possible though, otherwise an extension using these will have to also have the arbitration permission to accommodate all scenarios.

Reply:
Incorrect, my plan had been for the extension to transfer ownership to the user after submitting a stake. So the owner is in fact the user, while the extension is the one with administration permission.

Reply:
I guess I could believe that could work. The problem with that approach in my mind (in general, at least) is that the expenditure will then be very easily able to be altered out from 'underneath' the extension, which is going to make it hard to cover all edge cases on the extension 'level'.

Reply:
Can you elaborate? I was thinking of this extension as acting a bit like a rental service -- you put in a quarter and out pops an expenditure. Once you have the expenditure the extension takes a hard backseat, you do whatever you want, and then assuming everything ends well, you can go back and claim your deposit.

For other extensions (Task, etc) I was not planning on doing this -- in those cases, the owner would remain the extension. The difference is that in the former case, the extension provides no new functionality on top of the primitive, while in the latter cases it does. So that's the difference.

Reply:
> assuming everything ends well

It feels like there's a lot of functionality just being swept under the rug by this statement. Until there's a hard spec, this is just us getting sidetracked - and even if it works in the case of the extension you are thinking about, I am worrying about the general-purpose functionality of this primitive for all possible future extensions.

Either way, the way this decision needs to be made is from product - as I say, these changes are to make this primitive useful as a standalone object in their eyes, so the requirements need to come from them.

Reply:
I don't necessarily agree, but I'm comfortable putting this on hold until the staking extension is developed and releasing them together.